pkg/entities/set.go

```diff
@@ -189,3 +189,34 @@ func (s *set) createHeader(setType ContentType, templateID uint16) {
 		binary.BigEndian.PutUint16(s.headerBuffer[0:2], templateID)
 	}
 }
+
+// MakeTemplateSet is a convenience function which creates a template Set with a single Record.
+func MakeTemplateSet(templateID uint16, ies []*InfoElement) (*set, error) {
```

Comment:
Do we want to add these two functions to Set interface?

Reply:
These functions create a `Set` so I don't think we can add them to the interface?

Reply:
right, my brain is damaged